### ISSUE TITLE
Incorrect step in setting up LUIS recognizer in Appsettings

### ISSUE THREAD

**Issue by jwiley84** [MicrosoftDocs/bot-docs#1269]
In the section "Update the settings file", the portion that reads "Add the information required to access your LUIS app including application id, authoring key, and region into the appsettings.json file. These are the values you saved previously from your published LUIS app. Note that the API host name should be in the format &lt;your region&gt;.api.cognitive.microsoft.com." is actually incorrect.

What the user should be putting is JUST the region. Putting anything other then just the region results in 'No such host is known' error as seen [here](https://stackoverflow.com/questions/57105966/botbuilder-nlp-with-dispatch-error-no-such-host-is-known). A possible fix for the JS side of this is [here](https://github.com/microsoft/botbuilder-js/pull/1083)

![image](https://user-images.githubusercontent.com/17333688/61556151-0320e280-aa16-11e9-8e7c-635b826cd89f.png)

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 73b78c1c-bd0f-caec-32ad-47e6dfcc5b34
* Version Independent ID: 00957482-fbae-de4e-6c9c-49dae66c74af
* Content: [Add natural language understanding to your bot - Bot Service](https://docs.microsoft.com/en-us/azure/bot-service/bot-builder-howto-v4-luis?view=azure-bot-service-4.0&tabs=csharp)
* Content Source: [articles/v4sdk/bot-builder-howto-v4-luis.md](https://github.com/MicrosoftDocs/bot-docs/blob/live/articles/v4sdk/bot-builder-howto-v4-luis.md)
* Service: **bot-service**
* Sub-service: **cognitive-services**
* GitHub Login: @ivorb
* Microsoft Alias: **v-ivorb**

**Comment by zxyanliu** [comment]
Hi Jessica, 

Thanks for your report. I don't know if it will help but I'd like to share since I have been using LUIS to connect my local core bot recently. I followed this readme: 
https://github.com/microsoft/BotBuilder-Samples/tree/master/experimental/adaptive-dialog/csharp_dotnetcore/04.core-bot

I set the value for LuisAPIHostName in appsetting.json in this format: <your region>.api.cognitive.microsoft.com
For example, I filled in this: westus.api.cognitive.microsoft.com
And it works for me. 

Best, 

Yan
